.github/workflows/applicationsvalidation.yml

```
@@ -52,7 +54,7 @@ jobs:
         with:
           working-directory: ./framework
           run: >
-            ./mvnw -B clean verify 
+            ./mvnw -B package
```

Currently, tests are still executed. I don’t know whether the below also works for tycho-surefire

```suggestion
            ./mvnw -B package -Dmaven.test.skip=true
```

---

If I understand correctly, only Maven surefire tests are executed (i.e., tests in projects with the `.tests-without-platform` marker), as the plugin runs in the `test` phase, which is executed for the `package` goal. However, the Tycho surefire tests do not seem to executed, as they are run in the `integration-test` phase. Thus you proposal should be fine to also disable the Maven surefire tests.